comp/core/configsync/configsyncimpl/sync_integration_test.go

```diff
@@ -76,14 +76,14 @@ func TestRunWithChan(t *testing.T) {
 			require.EventuallyWithT(t, func(t *assert.CollectT) {
 				assert.EqualValues(t, i, callnb.Load())
 				assert.Equal(t, "not-value1", cs.Config.GetString("key1"))
-			}, 100*time.Millisecond, 10*time.Millisecond)
+			}, 200*time.Millisecond, 10*time.Millisecond)
```

Comment:
Is there a reason to not use 1 second as the condition will be retried every 10ms?

Reply:
Not really no, I just doubled the previous timeout 😄
It was already merged by the MQ before I saw your comment, but if it fails again I'll just increase it to 1s